language/go/fileinfo_test.go

```diff
@@ -57,7 +59,7 @@ func TestOtherFileInfo(t *testing.T) {
 
 			// Only check that we can extract tags. Everything else is covered
 			// by other tests.
-			if diff := cmp.Diff(tc.wantTags, got.tags, fileInfoCmpOption); diff != "" {
+			if diff := cmp.Diff(tc.wantTag, got.tags, fileInfoCmpOption); diff != "" {
```

Comment:
small nit: renaming this from plural to singular now made the comparison singular vs plural, hence a bit harder to read.